sdk/openai/tests.yml

```diff
@@ -0,0 +1,7 @@
+trigger: none
+
+extends:
+  template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
+  parameters:
+    ServiceDirectory: openai
+    SupportedClouds: 'Public'
```

Comment:
Do we need to restrict to a specific location?

Reply:
talked offline will leave it as is for now